react/features/base/conference/reducer.js

```diff
@@ -17,80 +17,98 @@ import {
     SET_AUDIO_ONLY,
     SET_DESKTOP_SHARING_ENABLED,
     SET_FOLLOW_ME,
+    SET_MAX_RECEIVER_VIDEO_QUALITY,
     SET_PASSWORD,
-    SET_RECEIVE_VIDEO_QUALITY,
+    SET_PREFERRED_RECEIVER_VIDEO_QUALITY,
     SET_ROOM,
     SET_SIP_GATEWAY_ENABLED,
     SET_START_MUTED_POLICY
 } from './actionTypes';
 import { VIDEO_QUALITY_LEVELS } from './constants';
 import { isRoomValid } from './functions';
 
+const DEFAULT_STATE = {
+    joining: undefined,
+    maxReceiverVideoQuality: VIDEO_QUALITY_LEVELS.HIGH,
+    preferredReceiverVideoQuality: VIDEO_QUALITY_LEVELS.HIGH
```

Comment:
Don't you wanna persist these?

Reply:
maxReceiverVideoQuality I don't think so, at least for now because that'll be toggled only when in tile view. I can see a case for persisting the user selected preferred device. I would even go as far as to say putting quality selection in the settings dialog would be good and maybe was discussed at one point.